edge/services/helloworld/MaintainHW.md

```diff
@@ -0,0 +1,18 @@
+# Process for the Horizon Development Team to Make Updates to the Helloworld Service
+
+Instructions for the open-horizon developers that are maintaining this example code:
+
+- Do the steps in the [README.md Preconditions](#preconditions) section, **except**:
+  - export `HZN_EXCHANGE_URL` to the staging instance
+  - Do **not** run `hzn dev service new ...` (use the git files in this directory instead)
+  - export `HZN_EXCHANGE_USER_AUTH` to your credentials in the IBM org
+- Make whatever code changes are necessary
+- Increment `SERVICE_VERSION` in `horizon/hzn.json`
+- Make `~/.hzn/keys/service.private.key` and `~/.hzn/keys/service.public.pem` actually be symbolic links to the common keys we use to sign all of our examples.
+- Build, test, and publish for all architectures:
+
+```bash
+make publish-all-arches
+```
+
+Note: building all architectures works on mac os x, and can be made to work on ubuntu via: http://wiki.micromint.com/index.php/Debian_ARM_Cross-compile , https://wiki.debian.org/QemuUserEmulation
```

Comment:
building s/b Building; should mac os be macOS?

Reply:
done